src/change/direct.rs

```diff
@@ -151,6 +151,15 @@ impl<'a> DirectApi<'a> {
         self.rtc.session.medias.last_mut().unwrap()
     }
 
+    /// Remove `Media`.
+    ///
+    /// Removes media and all streams belong to a media identified by a `mid`.
+    pub fn remove_media(&mut self, mid: Mid) {
+        self.rtc.session.medias.retain(|media| media.mid() != mid);
+        self.rtc.session.streams.remove_streams_by_mid(mid);
+        self.rtc.session.source_keys.retain(|_, v| v.0 != mid);
+    }
```

Comment:
Could we maybe move the remove_media call to `self.rtc.session` so that we don't need to `pub(crate)` the source_keys?